config/routes.rb

```diff
@@ -9,4 +9,17 @@
       resources :likes, only: [:index]
     end
   end
+  
+  namespace :api do
+    namespace :v1 do
+      get 'comments/index'
+      get 'comments/create'
+      get 'posts/index'
+      resources :users, only: [] do
+        resources :posts, only: [:index] do
+          resources :comments, only: %i[index create]
+        end
+      end
```

Comment:
Please ensure to rectify the problems with the API routes because when I try to access users, it throws the error seen in the screenshot below in the console, in Postman and in the frontend (after a user has been sign up for the frontend part). I recommend resolving this problem so the app can work smoothly as a service.

<img width="368" alt="image" src="https://github.com/osmanwako/blog_application/assets/84930355/f9525298-f09b-440c-b5aa-735669d8b70d">


<img width="410" alt="image" src="https://github.com/osmanwako/blog_application/assets/84930355/878852ad-1bb8-4cf4-b19b-2ff2ab7c1d37">


<img width="904" alt="image" src="https://github.com/osmanwako/blog_application/assets/84930355/29fcf395-1ad6-4faf-98de-e029592f5ca4">